aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -272,6 +275,14 @@ module aptos_token::token {
         expiration_sec: u64,
     }
 
+    /// capability to do partial withdraw with signer. The capability owner can withdraw up to amount token
+    struct PartialWithdrawCapability has drop, store {
```

Comment:
what happens if I create both a `WithdrawCapability` and a `PartialWithdrawCapability` for the same item?

Reply:
Yeah, looks weird. It seems the `WithdrawCapability` can include `PartialWithdrawCapability` and add a `withdraw_with_capability_v2` with an extra `amount` param. So why not just reuse `WithdrawCapability`?

Reply:
From the API and the fields they have. There is no reason to distinguish them IMO. allowing partial withdraw should be a subset of allowing full withdraw. I mean, if you allow ppl to withdraw in full, you should allow them to withdraw partially by default.

Reply:
They is real a `CapCap` PR. hahahahahahah

Reply:
updated, remove the redundant struct and only keep the new function